scripts/db_migrator.py

```diff
@@ -579,6 +579,23 @@ def migrate_port_qos_map_global(self):
             self.configDB.set_entry('PORT_QOS_MAP', 'global', {"dscp_to_tc_map": dscp_to_tc_map_table_names[0]})
             log.log_info("Created entry for global DSCP_TO_TC_MAP {}".format(dscp_to_tc_map_table_names[0]))
 
+    def migrate_route_table(self):
+        """
+        Handle route table migration. Migrations handled:
+        1. 'weight' attr in ROUTE object was introduced 202205 onwards.
+            Upgrade from older branch to 202205 will require this 'weight' attr to be added explicitly
+        """
+        route_table = self.appDB.get_table("ROUTE_TABLE")
+        for route_prefix, route_attr in route_table.items():
+            if 'weight' not in route_attr:
```

Comment:
Can you also check how the subnet routes looks like? IIRC, it may not have weights today.

Reply:
This logic will still add the weight attr to these routes.
It did not create an issue though, as the bootup path took two diff directions: For all other routes which do not have nexthops - weight attribute added by my logic will be removed after reconciliation.